package academy.devdojo.javacore.Oexception.exception.dominio;

import java.io.FileNotFoundException;

public class Funcioario extends  Pessoa{
    public void salvar() throws LoginInvalidoException,ArithmeticException {
        System.out.println("salvando funcioanrio");
    }
}
